utils/ci/check-tzdata-updates.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+set -e -x
+
+source default-config
+
+[[ -d "${WORKSPACE}/sources" ]] || die "Run get-sources.sh first"
+
+latest_tzdb_version=$(curl -s https://data.iana.org/time-zones/data/version);
+tzdb_version_in_repo=$(cat "${WORKSPACE}/sources/contrib/cctz/testdata/version");
+
+if [ "$tzdb_version_in_repo" = "$latest_tzdb_version" ];
+then
+    echo "No update for TZDB needed";
+    exit 0;
+else
+    echo "TZDB update required! Version in repo is ${tzdb_version_in_repo}, latest version is ${latest_tzdb_version}";
+    exit 1
+fi;
```

Comment:
smth like that can be added to ci, to ensure we never miss tzdb updates (even if cctz maintainers will miss that)